tlair/src/test/scala/at/forsyte/apalache/tla/lir/TestTlaExpr.scala

```diff
@@ -183,7 +184,7 @@ class TestTlaExpr extends AnyFunSuite {
       OperEx(TlaBoolOper.existsUnbounded, NameEx("x"), OperEx(TlaOper.eq, NameEx("x"), NameEx("x")))
     val ex2 =
       OperEx(TlaBoolOper.existsUnbounded, NameEx("x"), OperEx(TlaOper.eq, NameEx("x"), NameEx("x")))
-    val conj = OperEx(TlaBoolOper.and, ex1, ex2)
+    OperEx(TlaBoolOper.and, ex1, ex2)
```

Comment:
fruitless value

Reply:
So there is again no assert in many of these tests, I think they must just therefore be ensuring now crash happens when constructing the values? (Tho I'd honestly expect that to be statically guaranteed from the types, given that this is just building and AST...

So we actually need this value *just because* it uses ex1 and ex2 afaict.

Reply:
But it would probably be good to look at this test and decide if it's actually testing anything useful, or if it should get some asserts, be migrated to PBT, or maybe just binned entirely?